src/test/data/JsonSerializableAddressBookTest/duplicatePersonAddressBook.json

```diff
@@ -5,12 +5,14 @@
     "email": "alice@example.com",
     "address": "123, Jurong West Ave 6, #08-111",
     "insurancePlans" : "No added plans",
+    "claims" : "No claims yet.",
```

Comment:
maybe we could ensure same formatting? either include or exclude full stop

Reply:
I think we should exclude the full stop

Reply:
Good catch, fixed!